FAForever/downlords-faf-client
# Feature: Sound for match found

Added a new sound when the match-found notification triggers.
It is like a simple beep-beep-beep-beep race start sound, not too intrusive but clear.

- Added notification to enable/disable it (on by default).

- Added tests for sound playing as with other sounds.

Manual tests done:
✅  Check labels/checkboxes are ok in English.
✅  Check checkbox functions (on/off)
✅  Check sound plays when match is found
[ ] Check default settings show correctly in settings - notifications window (not sure how to reset settings...) 

Not sure how to add all the translations? Used the same words as for the "Match Found" string, so it should be easy to add these.

Fixes https://github.com/FAForever/downlords-faf-client/issues/2984
